internal/db/schema/migrations/generate/generate.go

```diff
@@ -118,23 +118,28 @@ func generate(dialect string) {
 		os.Exit(1)
 	}
 
-	outFile := fmt.Sprintf("%s/%s_migration.gen.go", baseDir, dialect)
+	outFile := fmt.Sprintf("%s/%s/%s_migration.gen.go", baseDir, dialect, dialect)
 	if err := ioutil.WriteFile(outFile, outBuf.Bytes(), 0o644); err != nil {
 		fmt.Printf("error writing file %q: %v\n", outFile, err)
 		os.Exit(1)
 	}
 }
 
 var migrationsTemplate = template.Must(template.Must(template.New("Content").Parse(
-	`{{ .Name }}: []byte(` + "`\n{{ .Content }}\n`" + `),
-`)).New("MainPage").Parse(`package schema
+	`{{ .Name }}: {
+		Statements: []byte(` + "`\n{{ .Content }}\n`" + `),
+	},
+`)).New("MainPage").Parse(`// Code generated by "make migrations"; DO NOT EDIT.
```

Comment:
Having this comment above the package makes it a package comment, but this is generated to a package where not every file is auto generated.

Reply:
Oh, sorry. I see now you moved it to its own package.